main.py

```diff
@@ -26,36 +29,265 @@ def info() -> Dict[str, List[str]]:
+
+
+@app.post("/upload/")
+async def create_upload_file(
+    request: Request,
+    image_id: UploadFile = File(...)
+):
+    if (
+        image_id.filename.endswith(".jpg") |
+        image_id.filename.endswith(".JPEG") |
+        image_id.filename.endswith(".jpeg") |
+        image_id.filename.endswith(".png") |
+        image_id.filename.endswith(".webp")
+    ):
+        with open("./app/static/imagenet_subset/n00000000_usersImage.JPEG", "wb") as buffer:
+            shutil.copyfileobj(image_id.file, buffer)
+            form = ClassificationForm(request)
+        await form.load_data()
+        image_id = "n00000000_usersImage.JPEG"
+        model_id = form.model_id
+        classification_scores = classify_image(
+            model_id=model_id, img_id=image_id)
+        request._url = URL("/classifications")
+        return templates.TemplateResponse(
+            "classification_output.html",
+            {
+                "request": request,
+                "image_id": image_id,
+                "classification_scores": json.dumps(classification_scores),
+                "backButton": "/delete"
+            },
+        )
+    else:
+        return templates.TemplateResponse(
+            "classification_select.html",
+            {
+                "request": request,
+                "images": list_images(),
+                "models": Configuration.models,
+                "userImage": 1
+            },
+        )
+
+
+@app.get("/delete")
+async def deleteFile(request: Request):
+    if (os.path.isfile("./app/static/imagenet_subset/n00000000_usersImage.JPEG")):
+        os.remove("./app/static/imagenet_subset/n00000000_usersImage.JPEG")
+    return templates.TemplateResponse(
+        "deleteFile.html",
+        {
+            "request": request,
+        }
+    )
+
+
+@app.get("/download_scores")
+async def download_scores():
+    return FileResponse('classification_scores.json', media_type='application/json', filename='classification_scores.json')
+
+
+@app.get("/download_plot")
+async def download_plot():
+    with open('classification_scores.json', 'r') as f:
+        classification_scores = json.load(f)
+
+    # Extract class labels and scores from the list
+    classes, scores = zip(*classification_scores)
+
+    # Create an index for each class
+    class_indices = list(range(1, len(classes) + 1))
+
+    # Invert the order of the classes
+    class_indices = class_indices[::-1]
+
+    # Create the appropriate dimension figure
+    plt.figure(figsize=(11, len(classes)))
+    # Create a bar plot
+    plt.grid(alpha=0.2)
+    plt.barh(class_indices, scores, color=[
+        '#1A4A04', '#750014', '#795703', '#06216C', '#3F0355'])
+
+    # Optionally rotate class labels for better readability
+    plt.yticks(class_indices, classes, ha='right')
+    plt.ylabel('Class')
+    plt.xlabel('Score')
+    plt.title('Classification Scores')
+
+    # Save the plot as a PNG file
+    plt.savefig('classification_plot.png')
```

Comment:
What happens if multiple users ask for a classification job? The file would be overwritten. Isn't it best to return the results directly? Or, if you reuse them later for other things, another way to do it would be to store the results with a unique name and clean up after the response is returned to the user (or after a while anyway).